requirements-devel.txt

```diff
@@ -11,7 +11,7 @@ click==8.1.7
 codespell==2.2.6
 colorama==0.4.6
 coverage==7.4.0
-craft-application @ git+https://github.com/canonical/craft-application@main
+craft-application @ git+https://github.com/canonical/craft-application@CRAFT-2273-check-base-destructive
```

Comment:
reminder to update this before merging

Reply:
can we use released craft-application now?